src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,77 @@
-import React from 'react';
+import cn from 'classnames';
+import {
+  FC,
+  ChangeEventHandler,
+  FormEventHandler,
+  useState,
+  memo,
+} from 'react';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ReponseError';
 import './FindMovie.scss';
+import { MovieCard } from '../MovieCard';
+import { MOVIE_DEFAULT_PREVIEW } from '../../constants/movies.constant';
+
+interface Props {
+  onMovieAdd: (movie: Movie) => void;
+}
+
+const NativeFindMovie: FC<Props> = ({ onMovieAdd }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+
+  const [query, setQuery] = useState('');
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+
+  const handleSubmit: FormEventHandler<HTMLFormElement> = (event) => {
+    event.preventDefault();
+
+    setIsLoading(true);
+
+    getMovie(query)
+      .then((response: MovieData | ResponseError) => {
+        if ('Error' in response) {
+          setError(response.Error);
+        } else {
```

Comment:
can you make it without `if / else` construction?

Reply:
Yes, I can use here `&&` operator instead of if / else statement, but I have linter error `ESLint: Expected an assignment or function call and instead saw an expression.(@typescript-eslint/no-unused-expressions)`. Maybe you can suggest something? 

![Screenshot_5](https://github.com/mate-academy/react_movies-list-fetch-movies/assets/114941446/e47a1fec-2b0e-43f2-ae39-0e00099654e9)

Reply:
Yes, I can do it with `&&` operator instead of if / else statement, but I got an Error `ESLint: Expected an assignment or function call and instead saw an expression.(@typescript-eslint/no-unused-expressions)`
![Screenshot_5](https://github.com/mate-academy/react_movies-list-fetch-movies/assets/114941446/c13eff13-cd91-4507-9538-e6f5c720e401)